src/main/java/cinema/model/Role.java

```diff
@@ -0,0 +1,45 @@
+package cinema.model;
+
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Role {
```

Comment:
are you supposed to create the private constructor here and the factory `of()` method?

Reply:
done